.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "stage": 1
```

Comment:
Yes, it works fine, https://github.com/ericelliott/prod-module-boilerplate/pull/35

Feel free to discard that PR if it is not practical to keep it in there.